content/docs/for-platform-operators/concepts/build-run-images.md

```diff
@@ -0,0 +1,94 @@
++++
+title="Base image types"
+weight=99
++++
+
+As you already know, `Cloud Native Buildpacks (CNBs)` transform your application source code into `OCI images`  that can run on any cloud and include all of the components your application needs to build and run and.
+
+<!--more-->
+
+Each buildpack checks the source code and provides any relevant dependencies. Then, a `base image` is then generated from the application’s source code and these dependencies.
```

Comment:
Is "base image" the correct term here?  We've used "application image" in the past to refer to the final output image.

Reply:
this is from the spec; not sure how valid it's

Reply:
The spec defines terminology at https://github.com/buildpacks/spec/blob/main/platform.md#terminology.

The terminology is defines *base image*, *build image*, *run image* and *app image*.  An *app image* a.k.a. *application image* is generated by combining the *run image* with the layers generated by executing a set of buildpacks against the application source code.

Reply:
updated already